web/src/pages/project/[projectId]/widgets/new.tsx

```diff
@@ -0,0 +1,592 @@
+import { useSession } from "next-auth/react";
+import { useRouter } from "next/router";
+import Page from "@/src/components/layouts/page";
+import {
+  Card,
+  CardContent,
+  CardDescription,
+  CardHeader,
+  CardTitle,
+} from "@/src/components/ui/card";
+import { HorizontalBarChart } from "@/src/features/widgets/chart-library/HorizontalBarChart";
+import { VerticalBarChart } from "@/src/features/widgets/chart-library/VerticalBarChart";
+import { VerticalBarChartTimeSeries } from "@/src/features/widgets/chart-library/VerticalBarChartTimeSeries";
+import { LineChartTimeSeries } from "@/src/features/widgets/chart-library/LineChartTimeSeries";
+import { PieChart } from "@/src/features/widgets/chart-library/PieChart";
+import { api } from "@/src/utils/api";
+import {
+  metricAggregations,
+  type QueryType,
+  mapLegacyUiTableFilterToView,
+} from "@/src/features/query";
+import { useState, useMemo, useEffect } from "react";
+import {
+  Select,
+  SelectContent,
+  SelectGroup,
+  SelectItem,
+  SelectLabel,
+  SelectTrigger,
+  SelectValue,
+} from "@/src/components/ui/select";
+import { Label } from "@/src/components/ui/label";
+import { viewDeclarations } from "@/src/features/query/dataModel";
+import { type z } from "zod";
+import { views } from "@/src/features/query/types";
+import { Input } from "@/src/components/ui/input";
+import { startCase } from "lodash";
+import { DatePickerWithRange } from "@/src/components/date-picker";
+import { InlineFilterBuilder } from "@/src/features/filters/components/filter-builder";
+import { useQueryFilterState } from "@/src/features/filters/hooks/useFilterState";
+import { useDashboardDateRange } from "@/src/hooks/useDashboardDateRange";
+import { type ColumnDefinition } from "@langfuse/shared";
+
+export default function NewWidget() {
+  const session = useSession();
+  const isAdmin = session.data?.user?.admin === true;
+
+  const router = useRouter();
+  const { projectId } = router.query as { projectId: string };
+
+  // State for form fields
+  const [widgetName, setWidgetName] = useState<string>("Traces");
+  const [widgetDescription, setWidgetDescription] = useState<string>(
+    "Traces grouped by name for the last 30 days.",
+  );
```

Comment:
ToDo: Review whether we want to set empty defaults and instead use placeholder to guide the user.